components/scream/src/physics/shoc/shoc_energy_fixer_impl.hpp

```diff
@@ -50,6 +50,8 @@ void Functions<S,D>::shoc_energy_fixer(
   Scalar se_dis = 0;
 
   // Compute linear interpolation of data into rho_zi
+  // This calculation is needed for stand alone tests,
+  // but it redundant when calling shoc_main
   linear_interp(team,zt_grid,zi_grid,rho_zt,rho_zi,nlev,nlevi,0);
```

Comment:
This can be removed if `shoc_energy_fixer()` is only called from `shoc_main` (and not the stand alone bfb test) since rho_zt is already interpolated in a previous function.